controllers/operator/containers.go

```diff
@@ -34,7 +34,7 @@ func buildInitContainers(mongoDBImage string) []corev1.Container {
 			Command: []string{
 				"bash",
 				"-c",
-				"until </dev/tcp/mongodb/27017 ; do sleep 5; done;",
+				"until </dev/tcp/common-service-db-ro/5432 ; do sleep 5; done;",
```

Comment:
Can we change this to readwrite service, `common-service-db-rw/5432` as this is not refusing the connection